code/modules/jobs/jobs.dm

```diff
@@ -52,8 +53,8 @@ GLOBAL_LIST_INIT(civilian_positions, list(
 GLOBAL_LIST_INIT(security_positions, list(
 	"Head of Security",
 	"Master-at-Arms",
-	"Detective",
-	"Security Officer"))
+	//"Redshield Officer",
+	"Security Officer")) //MAGISTRATE

```

Comment:
Either

//TODO: Magistrate

Or just remove the comment. TODO is a special snowflake thing.